content/en/security_chaos_engineering.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Security Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Security Chaos Engineering or SCE is a discipline of performing proactive security experimentation on a distributed system in order to build confidence in the System's capability to withstand turbulent and malicious conditions. Security Chaos Engineers use scientific method loop (steady-state, hypothesis, continuous verification, lesson learned, implement mitigation). This approach helps SOC Engineers, [Site Reliability Engineers](/site_reliability_engineering/), [DevOps](/devops/) and [DevSecOps](/devsecops/) Engineers to create confidence on the system, increase cyber resiliency and improve Security [Observability](/observability/).
```

Comment:
Couple questions:

- Is this practice only applicable to distributed systems?
- Do we need to define chaos engineering first?
- Does this definition also include broader operational concerns or is it only applicable to appsec?

Reply:
- yes it is mainly applicable to distributed systems, for just standalone apps, we can just run the classic regression tests, performance test etc...
- defined and linked 👍 
- yes, it includes infra-sec, app-sec, platform-sec and cloud/hybrid/multi-cloud-security

Reply:
changes made on this paragraph